### PR TITLE
Bolas update: Traitor items edition

### PR DESCRIPTION
**Pull request by PKPenguin321**
Adds reinforced bolas, they do a Weaken(1) in addition to normal bola functions and take twice as long to break out of when compared to normal ones. Their sprite has a different color.

Also added a message that displays to everybody when someone is ensnared by a bola.

Moves the throwing star bundle to uplink_kits.dm from weaponry.dm and makes it use a syndicate box.

Tweaked the Throwing Star box to be the Throwing Weapon box, updated uplink listing name and description accordingly. It now contains everything it used to in addition to two reinforced bolas.

Lowered throwing weapon bundle cost to 5 from 6 (it honestly should be lowered even further but I don't want to create controversy). At 6 points, it costed 1 less TC than an energy sword, while being significantly less useful.

Sprite is straight up taken from Paradise. Hopefully this is okay with @Fox-McCloud . I would have asked before PRing, but I don't really know any good way to get into contact.
bola

:cl: PKPenguin321
rscadd: Added reinforced bolas. They do a very short stun in addition to normal functions, and take twice as long to break out of when compared to regular bolas.
tweak: The traitor's Throwing Star Box has been made into the Throwing Weapons Box. It now contains two reinforced bolas in addition to its old contents. It also costs 5 TCs, down from 6.
/:cl:

### CODE REVIEW

**Comment by KazeEspada** [comment]
Instead of reinforced can they be energy bolas.

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -287,14 +287,25 @@
...
 		C << "<span class='userdanger'>\The [src] ensnares you!</span>"
+		if(stun)
+			C.Weaken(1)
+
+/obj/item/weapon/restraints/legcuffs/bola/tactical//traitor variant
+	name = "reinforced bola"
+	desc = "A strong bola, made with a long steel chain. It looks heavy, enough so that it could trip somebody."
+	icon_state = "bola_r"
+	breakouttime = 70
+	stun = 1
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/handcuffs.dm]: should be named weaken, and you could use it to set the weaken time as well.

**Comment by KorPhaeron** [comment]
Make them use the same sprite because meta

**Comment by Fox-McCloud** [comment]
>Sprite is straight up taken from Paradise. Hopefully this is okay with @Fox-McCloud . I would have asked before PRing, but I don't really know any good way to get into contact.

It's fine; we're AGPLv3--all I ask is that anything you port from us, you try to mention us, if possible--It really doesn't matter to me, to be honest---we utilize so much from you guys; I'm just happy to see some of our own stuff used elsewhere.

**Comment by Xhuis** [comment]
Throwing stars are insanely useful with a pneumatic cannon. Bolas could be even better. Load a pneumatic cannon with throwing stars and a reinforced bola - it's a guaranteed huge amount of damage, embeds, and a quick stun AND legcuff so they can't run away while you bludgeon them.

**Comment by Supermichael777** [comment]
can bolas embed?

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -513,12 +513,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
-/datum/uplink_item/stealthy_weapons/throwingstars
-	name = "Box of Throwing Stars"
-	desc = "A box of shurikens from ancient Earth martial arts. They are highly effective throwing weapons, \
-			and will embed into limbs when possible."
-	item = /obj/item/weapon/storage/box/throwing_stars
-	cost = 6
+/datum/uplink_item/stealthy_weapons/throwingweapons
+	name = "Box of Throwing Weapons"
+	desc = "A box of shurikens and reinforced bolas from ancient Earth martial arts. They are highly effective \
+			 throwing weapons. The bolas can knock a target down and the shirukens will embed into limbs."
```
Comment (RemieRichards) [inline code/modules/uplink/uplink_item.dm]: shuriken

**Comment by Xhuis** [comment]
@Supermichael777 Unless bolas have an embed chance anyway, no.

**Comment by PKPenguin321** [comment]
ughhh why does metastation have a box of throwing stars on the map

**Comment by Lumipharon** [comment]
probs random chance to spawn in the armoury locker.

Also why would you load bolas into a pneumatic when you can insta crit anyone just by loading it full of glass and shit?

And on another note, throwing stars are pretty gay purely because they're sneaky peaky throwing weapons, yet they create this HUGE TEXT saying who's throwing them, removing any chance of stealth (unlike suppressed guns which have no message).

**Comment by phil235** [comment]
>can bolas embed? (whith pnumatic cannon?)

If that's the case we have a bug, the throw_impact() code won't handle this correctly. :disappointed:

File: code/game/objects/items/weapons/handcuffs.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/items/weapons/handcuffs.dm]: does Weaken(0) do nothing? if so I can just make the var control the exact power of the weaken

**Comment by PKPenguin321** [comment]
pretty sure bolas can't embed

**Comment by PKPenguin321** [comment]
Ready to merge once 24 mark is hit, assuming travis passes

**Pull request merged**